card/mod/01_history/set/all/history.rb

```diff
@@ -217,15 +242,17 @@ def render_act act_view, args
 
   def render_action action_view, args
     action = args[:action] || card.last_action
-    hide_diff = Env.params["hide_diff"]=="true" || args[:hide_diff]
+    hide_diff = Env.params['hide_diff'] == ' true' || args[:hide_diff]
     name_diff =
       if action.card == card
         name_changes(action, hide_diff)
       else
         link_to name_changes(action, hide_diff),
-                path(view: :related, related: {view: "history",name: action.card.name}),
+                path(view: :related, related: {
+                  view: 'history',name: action.card.name
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Space missing after comma.